libsolidity/ast/Types.cpp

```diff
@@ -3228,6 +3225,18 @@ string MappingType::canonicalName() const
 	return "mapping(" + keyType()->canonicalName() + " => " + valueType()->canonicalName() + ")";
 }
 
+TypePointer MappingType::interfaceType(bool _inLibrary) const
+{
+	if (
+		!_inLibrary ||
+		!keyType()->interfaceType(_inLibrary) ||
```

Comment:
could call `canBeUsedExternally` - but I guess we will soon get rid of that anyway...